registry/remote/manifest.go

```diff
@@ -29,7 +29,8 @@ var defaultManifestMediaTypes = []string{
 	docker.MediaTypeManifestList,
 	ocispec.MediaTypeImageManifest,
 	ocispec.MediaTypeImageIndex,
-	artifactspec.MediaTypeArtifactManifest,
+	artifactspec.MediaTypeArtifactManifest, // TODO: deprecate
```

Comment:
same here

Reply:
Will do it after refactoring `Pack` and `ExtendedCopy`.